App/Components/GroupCard/index.tsx

```diff
@@ -13,7 +13,7 @@ interface ScreenProps {
   id: string
   name: string
   members: IContact[]
-  thumb?: IFiles
+  thumb?: string
```

Comment:
thumb is now just the hash of latest photo's thumb, that's all that was ever needed anyway